### PR TITLE
docs: (IAC-837) Add clarification text for V4_CFG_MANAGE_STORAGE settings

### PR DESCRIPTION
**Pull request by dhoucgitter**
This pull request supersedes PR #358, it has the same doc update content but it targets the staging branch instead of the stagingplus branch that PR #358 did.

### CODE REVIEW

**Review by jarpat — APPROVED** [review]

**Review by riragh — APPROVED** [review]

File: docs/CONFIG-VARS.md
Comment (thpang) [inline docs/CONFIG-VARS.md:264]: I would get clarification on the wording here. The last sentence seems a bit off. Have Susan or someone help with the wording.

File: docs/CONFIG-VARS.md
Comment (thpang) [inline docs/CONFIG-VARS.md:265]: Same as above.

File: docs/CONFIG-VARS.md
Comment (dhoucgitter) [reply docs/CONFIG-VARS.md:115]: How about adding this extra bit to help clarify what's creating the storage classes?
```suggestion
When V4_CFG_MANAGE_STORAGE is set to `true`, the efs-provisioner is deployed, the `sas` and `pg-storage` storage classes are created by the nfs-subdir-external-provisioner (EFS or NFS).
```

File: docs/CONFIG-VARS.md
Comment (dhoucgitter) [reply docs/CONFIG-VARS.md:264]: @supear, For the last sentence, I am trying to convey that if V4_CFG_MANAGE_STORAGE is set to false, viya4-monitoring will not create the default pg-storage storage class so the name of a pre-existing storage class must be provided. Can you suggest alternate wording to convey that more clearly than above?
Maybe I should also change the 2nd to last sentence to say: "The default pg-storage storage class is only created when `V4_CFG_MANAGE_STORAGE` is set to `true`.

**Comment by supear** [comment]
Proposed wording:   When `V4_CFG_MANAGE_STORAGE` is set to `true`, a storage class named `pg-storage` is created and is configured as the default storageClass. If `V4_CFG_MANAGE_STORAGE` is set to false, specify the name of an existing storage class for the value.

File: docs/CONFIG-VARS.md
Comment (dhoucgitter) [reply docs/CONFIG-VARS.md:119]: I would add the same suggestion as above to this GCP section and the Azure one above AWS.

File: docs/CONFIG-VARS.md
Comment (dhoucgitter) [reply docs/CONFIG-VARS.md:264]: @thpang here's that section with Susan's update, how's that sound now?
```suggestion
| postgres_storage_class | Storage class for the PostgreSQL PVCs | string | pg-storage | false |This value cannot be changed after the initial deployment. Supported for cadence versions 2022.10 and later. Only for internal databases. When `V4_CFG_MANAGE_STORAGE` is set to `true`, a storage class named `pg-storage` is created and is configured as the default storageClass. If `V4_CFG_MANAGE_STORAGE` is set to false, specify the name of an existing storage class for the value. | viya |
```

File: docs/CONFIG-VARS.md
Comment (dhoucgitter) [inline docs/CONFIG-VARS.md:111]:
```suggestion
When V4_CFG_MANAGE_STORAGE is set to `true`, the `sas` and `pg-storage` storage classes are created by the nfs-subdir-external-provisioner (Azure NetApp or NFS).
```

Comment (thpang) [reply docs/CONFIG-VARS.md:111]: Why are we identifying the tool set being used here? Not sure the advantage. It's now a maintenance item that would have to be checked and/or verified if we ever change. Just asking.

File: docs/CONFIG-VARS.md
Comment (thpang) [reply docs/CONFIG-VARS.md:119]: Same as above.

File: docs/CONFIG-VARS.md
Comment (thpang) [reply docs/CONFIG-VARS.md:265]: Was this one not changed or was it reviewed?

File: docs/CONFIG-VARS.md
Comment (supear) [reply docs/CONFIG-VARS.md:115]: How about this? When V4_CFG_MANAGE_STORAGE is set to `true`, the efs-provisioner is deployed and the `sas` and `pg-storage` storage classes are created (EFS or NFS). The storage classes are configured to use the same storage locations on the machines as the managed storage class (sas).

Comment (dhoucgitter) [reply docs/CONFIG-VARS.md:115]: @thpang, I misunderstood your comment, I've updated the suggested text to indicate that they both use the same storage location.

File: docs/CONFIG-VARS.md
Comment (dhoucgitter) [reply docs/CONFIG-VARS.md:111]: Wanted to align Azure with AWS and GCP sections, I've removed that reference as I misunderstood your earlier comment.

File: docs/CONFIG-VARS.md
Comment (dhoucgitter) [inline docs/CONFIG-VARS.md:265]: Same desired change, was just waiting for approval on line 264 before making the update to this item also. With this suggested change, they should match now.
```suggestion
| backrest_storage_class | Storage class for the pgBackrest PVCs | string | pg-storage | false |This value cannot be changed after the initial deployment. Supported for cadence versions 2022.10 and later. Only for internal databases. When `V4_CFG_MANAGE_STORAGE` is set to `true`, a storage class named `pg-storage` is created and is configured as the default storageClass. If `V4_CFG_MANAGE_STORAGE` is set to false, specify the name of an existing storage class for the value. | viya |
```

Comment (dhoucgitter) [reply docs/CONFIG-VARS.md:265]: @thpang I don't see the suggestion that you just resolved for line 264 in the commits. If you didn't also apply the suggested change for line 264 I will update 264 to match the same suggestion for line 265 below.

Comment (dhoucgitter) [reply docs/CONFIG-VARS.md:265]: committing this suggestion for line 265 and I will make the same change for line 264 that Thomas resolved above.

Comment (dhoucgitter) [reply docs/CONFIG-VARS.md]: @thpang lines 264 & 265 have now been updated with Susan's suggested text, please comment or resolve this one if you are good with that result.

File: docs/CONFIG-VARS.md
Comment (dhoucgitter) [reply docs/CONFIG-VARS.md:115]: @thpang, does the suggested change from Susan that I added for lines 111, 115 and 119 to say that the `pg-storage` class is using the same storage location as the `sas` storage class help make that clear? If so, you can commit those suggestions.

**Review by thpang — APPROVED** [review]
LGTM

**Pull request merged**
